src/test/java/org/kohsuke/github/GHAppTest.java

```diff
@@ -146,6 +146,27 @@ public void createToken() throws IOException {
         assertThat(installationToken2.getRepositories(), nullValue());;
     }
 
+    @Test
+    public void createTokenWithRepositories() throws IOException {
+        GHApp app = gitHub.getApp();
+        GHAppInstallation installation = app.getInstallationByUser("bogus");
```

Comment:
IDK if this is the preferred approach, but it matches the above `createToken` test (I ran it against a real app and everything though when generating the stubs and then just changed the names to match the other test).